moveit_msgs/DisplayTrajectory trajectory
bool gripperOpen
